bolt/test/AArch64/dw_cfa_gnu_window_save.test

```diff
@@ -1,8 +1,8 @@
-# Check that llvm-bolt can handle DW_CFA_GNU_window_save on AArch64.
+# Check that llvm-bolt refuses binaries with DW_CFA_GNU_window_save on AArch64.
 
 RUN: yaml2obj %p/Inputs/dw_cfa_gnu_window_save.yaml &> %t.exe
-RUN: llvm-bolt %t.exe -o %t.bolt 2>&1 | FileCheck %s
+RUN not: llvm-bolt %t.exe -o %t.bolt 2>&1 | FileCheck %s
```

Comment:
`not` is not a modifier of `RUN:` directive, there is a `not` helper binary built alongside other LLVM binaries - it "negates the result" of execution of the command line passed as its arguments (success vs. failure, but IIRC there is a `--crash` mode as well). Here, the command line is the result of expansion of `llvm-bolt %t.exe -o %t.bolt` - it is executed with `stdout` and `stderr` being passed through unchanged to `FileCheck`, but the return code of `llvm-bolt` is "negated".
```suggestion
RUN: not llvm-bolt %t.exe -o %t.bolt 2>&1 | FileCheck %s
```